pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[build-system]
+build-backend = "flit_core.buildapi"
+requires = ["flit_core >=3.8.0,<4"]
+
+[project]
+name = "UD_UM"
```

Comment:
@aryamccarthy to rename this to ud-compatibility (and thus name the package as such), we need to change the UD_UM folder to ud-compatibility. Is that okay with you?